### ISSUE TITLE
row disappears when navigate with the keyboard arrows

### ISSUE THREAD

**Issue by ideabox-thiago** [paulhodel/jexcel#1458]
At some point, when I navigate with the keyboard arrow with lazyLoading enabled, a row disappears from the visible area of the table. This issue can be checked on the lib's own page, https://bossanova.uk/jspreadsheet/v4/examples/lazy-loading.
Does anyone know any solution for this?

**Comment by GBonnaire** [comment]
i fail to reproduct your bug.

What do you use as a browser ? 

You navigate only top / down or left / right ?

**Comment by ideabox-thiago** [comment]
I navigate down / up. When the lib loads the next set of data, the problem occurs. I'm using Chrome browser.

**Comment by GBonnaire** [comment]
Yes, i see a problem, is just bad cursor position. When i have time, i make a PR for fix that. 

Maybe @pphod have already fix that.
